### DISCUSSION TITLE
Request for Implementing Advanced User Logging

### DISCUSSION THREAD

**Discussion by THF-Mediapics** [photoprism/photoprism#4633] [Q&A]
First of all, I want to express my sincere gratitude for the excellent work you’ve done with PhotoPrism. The software has become a fantastic tool for organizing and managing my photos.

I have a requirement that’s very important for my use case, and I’m hoping for your support. I would like to be able to track which user downloaded which photo from PhotoPrism and when. An advanced user logging feature like this would be extremely helpful for me to monitor activities within my PhotoPrism installation.

Therefore, my questions to you are:
	•	Is there already a feature or plugin that enables this kind of logging?
	•	Is such functionality planned for future versions or perhaps already in development?
	•	If not, would it be possible to consider this feature as a feature request?

I look forward to your responses and thank you in advance for your support.

**Comment by graciousgrey** [comment]
Thank you very much for your feedback!

We will consider adding more detailed information about downloads to the [audit logs](https://www.photoprism.app/pro/kb/audit-logs) in our [professional editions](https://www.photoprism.app/teams). Our [personal editions](https://www.photoprism.app/editions) do not currently include audit logs, also due to privacy considerations.
